chrome/manifest.json

```diff
@@ -20,5 +20,12 @@
   "browser_action": {
     "default_icon": "icon128.png",
     "default_title": "Report this site to webcompat.com"
-  }
+  },
+  "content_scripts": [
+    {
+      "matches": ["https://webcompat.com/"],
```

Comment:
If you wanna test this locally you can change this to (or add) "<all_urls>" and it will work on `localhost:5000` -- not sure why adding that to the matches list didn't want to work.